app/presenters/api/v2/measures/measure_legal_act_presenter.rb

```diff
@@ -0,0 +1,80 @@
+module Api
+  module V2
+    module Measures
+      class MeasureLegalActPresenter < SimpleDelegator
+        alias_method :regulation, :__getobj__
```

Comment:
This feels weird. Like we're exposing implementation detail of some python class :smile:. Not very ruby but :man_shrugging:

Reply:
Yeh - I should this - its a hangover from an earlier commit when I wasn't creating a custom initialize method